Product/configuration-management.md

```diff
@@ -439,129 +448,111 @@ The rule processor must produce one of these results (identified by the result's
-    "ref": ".x01",
-    "wght": 100
-  },
-  {
-    "id": "001@1.0.0",
-    "cfg": "1.0.0",
-    "ref": ".01",
-    "wght": 200
-  },
-  {
-    "id": "001@1.0.0",
-    "cfg": "1.0.0",
-    "ref": ".02",
-    "wght": 100
-  },
-  {
-    "id": "002@1.0.0",
-    "cfg": "1.0.0",
-    "ref": ".err",
-    "wght": 0
-  },
-  {
-    "id": "002@1.0.0",
-    "cfg": "1.0.0",
-    "ref": ".x01",
-    "wght": 100
-  },
-  {
-    "id": "002@1.0.0",
-    "cfg": "1.0.0",
-    "ref": ".x02",
-    "wght": 100
-  },
-  {
-    "id": "002@1.0.0",
-    "cfg": "1.0.0",
-    "ref": ".01",
-    "wght": 100
-  },
-  {
-    "id": "002@1.0.0",
-    "cfg": "1.0.0",
-    "ref": ".02",
-    "wght": 200
-  }
-]
+```JSON
+    "rules": [
+        {
+            "id": "001@1.0.0",
+            "cfg": "1.0.0",
+            "termId": "v001at100at100",
+            "wghts": [
+                {
+                    "ref": ".err",
+                    "wght": 0
+                },
+                {
+                    "ref": ".x00",
+                    "wght": 100
+                },
+                {
+                    "ref": ".x01",
+                    "wght": 100
+                },
+                {
+                    "ref": ".01",
+                    "wght": 0
+                },
+                {
+                    "ref": ".02",
+                    "wght": 200
+                },
+                {
+                    "ref": ".03",
+                    "wght": 300
+                },
+            ]
+        },
+        {
+            "id": "002@1.0.0",
+            "cfg": "1.0.0",
+            "termId": "v002t100at100",
+            "wghts": [
+                {
+                    "ref": ".err",
+                    "wght": 0
+                },
+                {
+                    "ref": ".00",
+                    "wght": 0
+                },
+                {
+                    "ref": ".01",
+                    "wght": 0
+                },
+                {
+                    "ref": ".02",
+                    "wght": 1
+                },
+                {
+                    "ref": ".03",
+                    "wght": 0
+                }
+            ],
+        }
+    ]
 ```
 
 ### The expression object
 
-The expression object in the typology processor defines the formula that is used to calculate the typology score. The expression is able to accommodate any formula composed out of a combination of multiplication (`*`), division (`/`), addition (`+`) and subtraction (`-`) operations.
+The expression object in the typology processor defines the formula that is used to calculate the typology score. The expression is able to accommodate any formula composed out of a combination of multiplication (`multiply`), division (`divide`), addition (`add`) and subtraction (`subtract`) operations.  The expression object uses an  abbreviated [MathJSON](https://cortexjs.io/math-json/) _<sup>L</sup><sub>A</sub><sup>T</sup><sub>E</sub><sup>X</sup>_ format. 
```

Comment:
It seems that the keywords to invoke MathJSON is capitalised, e.g. "Multiply". It may be worth listing them like that here. Maybe also put them in quotes: "`Multiply`".

Reply:
Updated